### ISSUE TITLE
Disable chat commands

### ISSUE THREAD

**Issue by Matthww** [LegendOfMCPE/EssentialsPE#397]
#### Description

Disable chat commands like /tell or /me whenever a person is muted

#### Versions

* PocketMine Version: api3/block commit: 83471aa
* EssentialsPE Version: 065949e

#### Crashdump or console errors

```
```

**Comment by Sandertv** [comment]
Yep, will be working on that in the EssentialsPE 3.0.0 rewrite. The structure of the current EssentialsPE is not ideal to implement this.

**Comment by HimbeersaftLP** [comment]
If you add this, I would recommend setting a config option for that because some server owners may not like that....

**Comment by Sandertv** [comment]
Yep, that's a good idea @HimbeersaftLP.

**Comment by Matthww** [comment]
Config where you can disable a specific command if you are muted
